spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/RecurrentDecoder.scala

```diff
@@ -121,18 +130,21 @@ class RecurrentDecoder[T : ClassTag](seqLength: Int)
 
   override def backward(input: Tensor[T], gradOutput: Tensor[T]): Tensor[T] = {
     val st = System.nanoTime
+    gradInput.resizeAs(output)
```

Comment:
gradInput should be the same size as input; the gradInput of RecurrentDecoder is the gradInput of the first cell, no?

Reply:
In RecurrentDecoder, hiddensize is the same as inputsize and input doesn't have time dim while output and gradInput has. So i resize gradInput as output.